bokehjs/src/lib/models/layouts/layout_dom.ts

```diff
@@ -242,12 +242,21 @@ export abstract class LayoutDOMView extends DOMView {
     super.connect_signals()
 
     if (this.is_root)
-      window.addEventListener("resize", () => this.resize())
+      window.addEventListener("resize", this)
 
     // XXX: this.connect(this.model.change, () => this.layout())
     this.connect(this.model.properties.sizing_mode.change, () => this.layout())
   }
 
+  handleEvent(): void {
```

Comment:
@dracorpg, one thing to consider in future is to make the class implementing this method declare the interface being implemented (`EventListenerObject` in this case). Given that we use snake-case naming convention, one could be eager to rename this method accordingly, or even remove it, as it isn't referenced (at least directly) anywhere. I implemented this in https://github.com/bokeh/bokeh/pull/8085/commits/9e776b491d1eda1c9fdf907d5f630d57e3357ea9.